### PR TITLE
Implements Search function in Shop for players

### PR DESCRIPTION
**Pull request by ghost**
May be some code can be optimized, if so do not hesitate to modify before merging :)

### CODE REVIEW

File: app/Http/Controllers/ShopController.php [hunk 1]
```diff
@@ -36,6 +36,34 @@ public function index()
           'count'           => Cart::count()
         ]);
     }
+    
+    /**
+     * POST /shop/search
+     */
+    public function search(Request $request)
+    {
+		if(empty($request->input('search_value')))
```
Comment (mathieuletyrant) [inline app/Http/Controllers/ShopController.php]: Please, use {} with your if statement.

File: app/Http/Controllers/ShopController.php [hunk 2]
```diff
@@ -36,6 +36,34 @@ public function index()
...
+			return redirect(route('shop'))->with('error', Lang::get('flashMessage.shop.no_search_empty'));
+		
+		$accountLevel = AccountLevel::where('account_id', '=', Session::get('user.id'))->first();
```
Comment (mathieuletyrant) [inline app/Http/Controllers/ShopController.php]: Please, check your indentation.

File: resources/lang/fr/all.php [hunk 3]
```diff
@@ -43,7 +43,10 @@
         'categories'         => 'Catégories',
         'categories_desc'    => 'Sous-catégories disponibles',
         'summary'            => 'Récapitulatif',
-        'no_characters'      => "Vous n'avez pas de personnage sur le serveur"
+        'no_characters'      => "Vous n'avez pas de personnage sur le serveur",
+        'search_title'       => "Chercher un item",
+        'search_btn'  	     => "Chercher",
+        'search_result'	     => "Résultat de la recherche"
```
Comment (mathieuletyrant) [inline resources/lang/fr/all.php]: Wrong indentation

File: resources/lang/fr/flashMessage.php [hunk 4]
```diff
@@ -8,7 +8,8 @@
         'fail_category_id' => "La catégorie est surement vide ou n'existe pas",
         'empty_cart'       => "Votre panier est vide. Vous ne pouvez pas accéder à cette page",
         'not_shop_points'  => "Vous n'avez pas assez de point boutique sur votre compte",
-        'success'          => "Achat effectué, tapez la commande : .shop IG sur le personnage "
+        'success'          => "Achat effectué, tapez la commande : .shop IG sur le personnage ",
+		'no_search_empty'  => "Veuillez indiquer un élément de recherche"
```
Comment (mathieuletyrant) [inline resources/lang/fr/flashMessage.php]: Wrong indentation

File: resources/lang/en/all.php [hunk 5]
```diff
@@ -43,7 +43,10 @@
         'categories'         => 'Categories',
         'categories_desc'    => 'Sub-categories available',
         'summary'            => 'Summary',
-        'no_characters'      => 'You do not have character on the server'
+        'no_characters'      => 'You do not have character on the server',
+        'search_title'       => "Find an item",
+        'search_btn'  	     => "Search",
```
Comment (mathieuletyrant) [inline resources/lang/en/all.php]: Wrong indentation

File: resources/lang/en/all.php [hunk 6]
```diff
@@ -43,7 +43,10 @@
...
+        'search_result'	     => "Search result"
```
Comment (mathieuletyrant) [inline resources/lang/en/all.php]: Wrong indentation

File: resources/lang/en/flashMessage.php [hunk 7]
```diff
@@ -8,7 +8,8 @@
         'fail_category_id' => "Empty category",
         'empty_cart'       => "Empty Cart",
         'not_shop_points'  => "You have not enough shop points on your account",
-        'success'          => "Successful purchase : Write .shop IG on the character"
+        'success'          => "Successful purchase : Write .shop IG on the character",
+		'no_search_empty'  => "Search field cannot be empty"
```
Comment (mathieuletyrant) [inline resources/lang/en/flashMessage.php]: Wrong indentation

**Review by mathieuletyrant — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Done

**Pull request merged**